app/views/api/projects/work_times.jbuilder

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+json.project do
```

Comment:
do we need this ? Since we're on the project view already does this endpoint need to return project information again ?

Reply:
We do not have any kind of global store in frontend, so project view has no information about project before rendering. I've decided it is better to include this info in one response, than to make second request.